node/network/protocol/src/lib.rs

```diff
@@ -438,6 +438,62 @@ pub mod v1 {
 	impl_try_from!(CollationProtocol, CollatorProtocol, CollatorProtocolMessage);
 }
 
+/// Unified annoyance cost and good behavior benefits.
+#[derive(Debug,Clone,Copy)]
+#[allow(missing_docs)]
+pub enum UnifiedReputationChange {
+	MajorAnnoy(&'static str),
```

Comment:
nit: I'd personally prefer Cost instead of Annoy. Curious what @eskimor thinks

Reply:
The naming was derived from the naming in the issue. I am happy with either. @rphmeier